diff -Naur puremagic-1.25.old/setup.py puremagic-1.25/setup.py
--- puremagic-1.25.old/setup.py	2024-06-22 18:35:35.932247175 +0200
+++ puremagic-1.25/setup.py	2024-06-22 18:36:29.481874736 +0200
@@ -7,7 +7,7 @@
 
 root = os.path.abspath(os.path.dirname(__file__))
 
-with open(os.path.join(root, "puremagic", "main.py")) as reuse_file:
+with open(os.path.join(root, "puremagic", "main.py", encoding="utf-8")) as reuse_file:
     reuse_content = reuse_file.read()
 
 attrs = dict(re.findall(r"__([a-z]+)__ *= *['\"](.+)['\"]", reuse_content))
